src/Microsoft.SqlTools.ServiceLayer/Migration/MigrationService.cs

```diff
@@ -344,27 +344,7 @@ public void InitializeService(ServiceHost serviceHost)
                     prefs.EligibleSkuCategories = GetEligibleSkuCategories("AzureSqlDatabase", parameters.IncludePreviewSkus);
                     resultSet.sqlDbResults = provider.GetSkuRecommendation(prefs, req);
 
-                    if (resultSet.sqlDbResults.Count < parameters.DatabaseAllowList.Count)
-                    {
-                        // if there are fewer recommendations than expected, find which databases didn't have a result generated and create a result with a null SKU
-                        List<string> databasesWithRecommendation = resultSet.sqlDbResults.Select(db => db.DatabaseName).ToList();
-                        foreach (var databaseWithoutRecommendation in parameters.DatabaseAllowList.Where(db => !databasesWithRecommendation.Contains(db)))
-                        {
-                            resultSet.sqlDbResults.Add(new SkuRecommendationResult()
-                            {
-                                //SqlInstanceName = sqlDbResults.FirstOrDefault().SqlInstanceName,
-                                SqlInstanceName = parameters.TargetSqlInstance,
-                                DatabaseName = databaseWithoutRecommendation,
-                                TargetSku = null,
-                                MonthlyCost = null,
-                                Ranking = -1,
-                                PositiveJustifications = null,
-                                NegativeJustifications = null,
-                            });
-                        }
-                    }
-
-                    sqlDbStopwatch.Stop();
```

Comment:
looks like you removed the stopwatch `Stop()` call here by accident, we'll need that back